ts/src/tokocrypto.ts

```diff
@@ -1616,14 +1618,20 @@ export default class tokocrypto extends Exchange {
         //
         if (uppercaseType === 'MARKET') {
             const quoteOrderQty = this.safeValue (this.options, 'quoteOrderQty', true);
+            if (this.options['createMarketBuyOrderRequiresPrice'] && side === 'buy') {
```

Comment:
@samgermain this is wrong because you should only throw if the user did not provide a price or a cost, but you don't; check that before throwing the error